web/src/sidebar_ui.ts

```diff
@@ -133,13 +133,9 @@ export function update_invite_user_option(): void {
 
 export function update_unread_counts_visibility(): void {
     const hidden = !user_settings.web_left_sidebar_unreads_count_summary;
-
-    const $channel_sections: JQuery = $(".stream-list-subsection-header");
-    const $home_view_li: JQuery = $(".top_left_row");
-
-    for (const $el of [$home_view_li, $channel_sections]) {
-        $el.toggleClass("hide-unread-messages-count", hidden);
-    }
+    $(".top_left_row").toggleClass("hide-unread-messages-count", hidden);
```

Comment:
So I notice that this selector is wrong -- it should be just `.selected-home-view`, right? Or maybe all the home views... but this applies to all VIEWS views, for some reason.

Reply:
No functional effect, though, since we only do something with that class within `selected-home-view`. We can investigate this later.

Reply:
Oh yeah, that would be better. I wasn't paying close attention to it, but I think you're right

Reply:
The thing I'm not sure about is whether it's important if we change the home view that this got tagged on the other home view. Might be that what gets toggled should just be a class on the VIEWS parent element or something.